public/src/components/channelManagement/headerTests/headerTestVariantEditor.tsx

```diff
@@ -219,6 +246,39 @@ const HeaderTestVariantEditor: React.FC<HeaderTestVariantEditorProps> = ({
           deviceType={variant.mobileContent === undefined ? 'ALL' : 'NOT_MOBILE'}
         />
       </div>
+
+      <RadioGroup
+        value={variant.mobileContent !== undefined ? 'enabled' : 'disabled'}
+        onChange={onMobileContentRadioChange}
+      >
+        <FormControlLabel
+          value="disabled"
+          key="disabled"
+          control={<Radio />}
+          label="Show the same copy across devices"
+          disabled={!editMode}
+        />
+        <FormControlLabel
+          value="enabled"
+          key="enabled"
+          control={<Radio />}
+          label="Show different copy on mobile"
+          disabled={!editMode}
+        />
+      </RadioGroup>
+      {variant.mobileContent && (
+        <HeaderTestVariantContentEditor
```

Comment:
I've just realised that we only show the primary cta on mobile. So the tool should not show fields for heading/subheading + 2nd cta

Reply:
I've pushed a commit for this